.travis.yml

```diff
@@ -2,4 +2,3 @@ language: node_js
 node_js:
   - '10'
   - '8'
-  - '6'
```

Comment:
Is there a reason why 6 is being dropped in this PR? Otherwise, it should be its own PR, please.

Reply:
Sorry, my fault.